com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/visitor/CobolVisitor.java

```diff
@@ -117,9 +119,35 @@ public Class visitProcedureSectionHeader(ProcedureSectionHeaderContext ctx) {
     return visitChildren(ctx);
   }
 
+  @Override
+  public Class visitWorkingStorageSection(WorkingStorageSectionContext ctx) {
+    outlineTreeBuilder.addNodeToParent("WORKING-STORAGE SECTION", NodeType.SECTION, ctx);
+    outlineTreeBuilder.initVariables();
+    return visitChildren(ctx);
+  }
+
+  @Override
+  public Class visitDataDescriptionEntryFormat2(DataDescriptionEntryFormat2Context ctx) {
+    outlineTreeBuilder.addVariable(66, ctx.dataName1().getText(), NodeType.FIELD_66, ctx);
```

Comment:
optional: we could extract the number associated with the enum field 66, ... since will never change